.github/workflows/labeler.yml

```diff
@@ -10,7 +10,10 @@ jobs:
     steps:
 
     - uses: actions/labeler@main
-      if: github.event.pull_request.merged == true
+      if: >
+        github.event.pull_request.merged == true &&
+        !startsWith(github.event.pull_request.title, 'chore(deps):') &&
+        !startsWith(github.event.plul_request.title, 'ci(fix):'
```

Comment:
looks like a typo: `plul_request`

Reply:
Yes, it didn’t work on the next PR so I fixed it. It’s working now.